styles.css

```diff
@@ -229,8 +257,12 @@ h4 {
   overflow: hidden;
 }
 
-.content-section h2 {
-  color: #6e979e;
+.content-section li+li{
+  margin-top: 0.5em;
+}
+
+.content-section *+h2 {
```

Comment:
Any `h2` following something else (not first child) will have a bit more spacing than default for better legibility